README.md

```diff
@@ -1,10 +1,93 @@
 # Aind.Behavior.Pirouette
 
-A repository with source code for long-term, freely moving, electrophisiology recordings in mice
+Authors: Bruno Cruz, Brandon Pratt, and Carl Schoonover
```

Comment:
I generally avoid adding this to the readme since the repository already keeps track of contributors. If you want to add Carl, he just has to push a single commit, or you can add him as a co-author in one of them.